components/library/tntdb/Makefile

```diff
@@ -0,0 +1,72 @@
+#
+# This file and its contents are supplied under the terms of the
+# Common Development and Distribution License ("CDDL"), version 1.0.
+# You may only use this file in accordance with the terms of version
+# 1.0 of the CDDL.
+#
+# A full copy of the text of the CDDL should have accompanied this
+# source.  A copy of the CDDL is also available via the Internet at
+# http://www.illumos.org/license/CDDL.
+#
+
+#
+# Copyright 2016 Jim Klimov
+#
+
+# BuildRequires: cxxtools, sqlite-3, pg_config (postgres-9*/developer),
+# mysql_config (mariadb-55/client, percona, mysql)
+
+include ../../../make-rules/shared-macros.mk
+
+COMPONENT_NAME=	tntdb
+COMPONENT_VERSION=	1.3
+IPS_COMPONENT_VERSION=	1.3.0
+COMPONENT_SUMMARY=	C++ library to access SQL databases, easy and robust
+COMPONENT_SRC= $(COMPONENT_NAME)-$(COMPONENT_VERSION)
+COMPONENT_ARCHIVE= $(COMPONENT_SRC).tar.gz
+COMPONENT_ARCHIVE_HASH= \
+  sha256:093ab2694bc66bc05ae0a5c8a505d75ae84d4c35c2ddc0ff0283c5b9574b474b
+COMPONENT_ARCHIVE_URL= \
+  http://www.tntnet.org/download/$(COMPONENT_ARCHIVE)
+COMPONENT_PROJECT_URL=	http://www.tntnet.org/tntdb.html
+COMPONENT_FMRI= library/$(COMPONENT_NAME)
+COMPONENT_CLASSIFICATION= System/Libraries
+COMPONENT_LICENSE = LGPLv2.1
+COMPONENT_LICENSE_FILE = COPYING
+
+include $(WS_TOP)/make-rules/prep.mk
+include $(WS_TOP)/make-rules/configure.mk
+include $(WS_TOP)/make-rules/ips.mk
+
+CXXFLAGS.32=-D_FILE_OFFSET_BITS=64 -D_LARGEFILE_SOURCE
+CFLAGS.32=-D_FILE_OFFSET_BITS=64 -D_LARGEFILE_SOURCE
+CXXFLAGS+=$(CXXFLAGS.$(BITS))
+CFLAGS+=$(CFLAGS.$(BITS))
+
+# We use a fixed default mysql implementation, and a baseline
+# postgres version is defined in make-rules/*.mk above
+MYSQL_BINDIR.32=/usr/mariadb/5.5/bin
```

Comment:
Maybe, but rather as another PR since this would touch a few other components that use MySQL - better fix all consumers consistently and in one blow.